geniesp/bpc_redcap_export_mapping.py

```diff
@@ -7,17 +7,39 @@
   Add headers
   REMOVE PATIENTS/SAMPLES THAT DON'T HAVE GENIE SAMPLE IDS
 """
-from abc import ABCMeta, abstractmethod
+from abc import ABCMeta
+from datetime import date
 import math
 import os
 import subprocess
 
+import genie
+from genie import create_case_lists, process_functions
+import pandas as pd
 import synapseclient
 from synapseclient import File, Folder
-import pandas as pd
 
-import genie
-from genie import create_case_lists, process_functions
+from . import metafiles
+
+
+CBIO_FILEFORMATS_ALL = [
+    "data_timeline_treatment.txt",
+    "data_timeline_cancer_diagnosis.txt",
+    "data_timeline_pathology.txt",
+    "data_timeline_sample_acquisition.txt",
+    "data_timeline_medonc.txt",
+    "data_timeline_imaging.txt",
+    "data_timeline_sequencing.txt",
+    "data_timeline_labtest.txt",
+    "data_clinical_supp_survival.txt",
+    "data_clinical_supp_survival_treatment.txt",
+    "data_clinical_sample.txt",
+    "data_clinical_patient.txt",
+    "data_mutations_extended.txt",
+    "data_cna_hg19.seg",
```

Comment:
Currently this file is an issue because for some reason the seg files have always been created like

```
genie_{cohort}_data_cna_hg19.seg
```

Since it changes from cohort to cohort, this has become an issue as it seems i have to append it to the list of files to generate metadata file for.

Reply:
Unfortunate.  But looks like the hack provides a good work-around for now.